assets/app/features/upgrade/controllers/upgrade-upgrade-administration-server.controller.js

```diff
@@ -24,6 +24,14 @@
             checkAdminUpgrade: checkAdminUpgrade
         };
 
+        activate();
+
+        function activate() {
+            // On page load we need to check to see if the upgrade is already running to set
+            // the button status and the update check running
+            checkAdminUpgrade();
```

Comment:
Don't remove it, just add a comment saying it will be properly implemented and tested as part of that card